src/style.css

```diff
@@ -0,0 +1,225 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  src: url(./fonts/Avenir-Heavy.ttf);
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin-top: 20px;
+}
+
+.input-big {
+  height: 70px;
+  width: 100%;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-big:hover {
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input-big:focus {
+  box-sizing: border-box;
+  position: relative;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input-big::placeholder {
+  position: relative;
+  left: 0;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4d61;
+}
+
+.input-big::placeholder:hover {
+  position: relative;
+  left: 1px;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input-big::placeholder:focus {
+  position: relative;
+  left: 1px;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.big-icon {
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.big-icon:hover {
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.big-icon:focus {
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.input-small {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-small:hover {
+  width: 100%;
+  height: 42px;
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
```

Comment:
The same here. Most rules are duplicates of main `.input-small` rules. You should just pass the rules which are different than rules in main selector.
So here you need just `box-shadow` rule.